OWNERS

```diff
@@ -14,5 +14,5 @@ reviewers:
 - dthomson25
 - tetchel
 - wtam2018
-- ishitasequeira
```

Comment:
Looks like ishitasequeira was accidently removed. Please revert this line removal